dot/network/spam_limiter.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package network
+
+import (
+	"sync"
+	"time"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	lrucache "github.com/ChainSafe/gossamer/lib/utils/lru-cache"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+const MaxTimeWindow = 1 * time.Minute
```

Comment:
is this value used?

Reply:
Yep, in the [config](https://github.com/ChainSafe/gossamer/pull/4219/files#diff-4270f10fa6f73763776f65731c1b428543d6317e932d7bae6ebffa4682330f02R162)